data/software.json

```diff
@@ -954,18 +937,6 @@
         "platforms": [],
         "url": null
     },
-    {
-        "categories": [
-            "server"
-        ],
-        "doap": null,
-        "name": "Jerry Messenger",
```

Comment:
I understand the preference to delete unmaintained projects. However, I wonder if we should keep the for archive reasons. XMPP has a history, I would not like to just delete it.

Reply:
I don't see a reason in keeping dead projects on the list. What would be the benefit? There is an internet archive for things which don't exist anymore. I don't think this list's purpose is to archive _every_ XMPP software there was in history. Instead, it should list _available_ software.